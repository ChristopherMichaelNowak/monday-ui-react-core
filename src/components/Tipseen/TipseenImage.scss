@import "../../styles/themes.scss";
@import "../../styles/typography.scss";
@import "../../styles/global-css-settings.scss";

.monday-style-tipseen-image {
  object-fit: cover;
  border: solid 2px;
  @include theme-prop(border-color, primary-color);
  width: 100%;
  border-radius: $border-radius-medium;
  position: relative;
  z-index: 1;
}