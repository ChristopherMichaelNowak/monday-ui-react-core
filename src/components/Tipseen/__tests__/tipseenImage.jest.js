import React from "react";
import renderer from "react-test-renderer";
import TipseenImage from "../TipseenImage";

jest.useFakeTimers();

describe("Tipseen image tests", () => {
  describe("Snapshot Tests", () => {
    it("renders correctly", () => {
      const tree = renderer
        .create(
          <TipseenImage src="fake-tipseen-src" alt="alternative text" className="mock-class-name">
            <div />
          </TipseenImage>
        )
        .toJSON();
      expect(tree).toMatchSnapshot();
    });
    it("renders correctly with empty props", () => {
      const tree = renderer.create(<TipseenImage />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
