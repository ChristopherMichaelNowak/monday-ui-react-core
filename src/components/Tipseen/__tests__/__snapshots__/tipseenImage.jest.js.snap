// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tipseen image tests Snapshot Tests renders correctly 1`] = `
<img
  alt="alternative text"
  className="monday-style-tipseen-image mock-class-name"
  src="fake-tipseen-src"
/>
`;

exports[`Tipseen image tests Snapshot Tests renders correctly with empty props 1`] = `
<img
  className="monday-style-tipseen-image"
/>
`;
